tangelo/algorithms/variational/adapt_vqe_solver.py

```diff
@@ -93,6 +94,7 @@ def __init__(self, opt_dict):
         self.deflation_circuits: Optional[List[Circuit]] = copt_dict.pop("deflation_circuits", list())
         self.deflation_coeff: float = copt_dict.pop("deflation_coeff", 1)
```

Comment:
could we use `1.` as the default value, to make this clearly a float in the code ?